src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -93,6 +93,45 @@ class CallCredentials(object):
   def __init__(self, low_credentials):
     self._low_credentials = low_credentials
 
+class GoogleOAuth2MetadataPlugin(interfaces.GRPCAuthMetadataPlugin):
+  """ A metadata plugin supporting Google OAuth2 authentication for use with 
+  the oauth2client module.
+  """
+  
+  def __init__(self, credentials):
```

Comment:
I think it would make sense to refer to credentials coming from `oauth2client_client.GoogleCredentials.get_application_default()` as googleCredentials. That convention would give some extra transparency to the code.